src/Resources/contao/pages/PageRegular.php

```diff
@@ -88,6 +88,7 @@ protected function prepare($objPage)
 
 		/** @var ThemeModel $objTheme */
 		$objTheme = $objLayout->getRelated('pid');
+		\System::getContainer()->get('contao.image.picture_factory')->setDefaultDensities($objTheme->defaultImageDensities);
```

Comment:
I don’t think so, because if the default densities are already set in the picture factory, we want to reset it to `''` if the theme doesn’t have it configured.